apps/ui/src/helpers/argentx.ts

```diff
@@ -43,8 +43,11 @@ export default class Connector extends LockConnector {
 
   async logout() {
     const argentx = await get();
-    await argentx.disconnect({
-      clearLastWallet: true
-    });
+
+    try {
+      await argentx.disconnect({
+        clearLastWallet: true
+      });
+    } catch (e) {}
```

Comment:
I think we should still log in console in case of error. just in case if there is some other error.

Reply:
```suggestion
    } catch (e) {
      console.log(e)
    }
```